src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/commands.py

```diff
@@ -0,0 +1,119 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._util import (get_monitor_alert_rules_operation, get_monitor_alert_rule_incidents_operation,
+                    get_monitor_log_profiles_operation, get_monitor_autoscale_settings_operation,
+                    get_monitor_service_diagnostic_settings_operation,
+                    get_monitor_usage_metrics_operation, get_monitor_event_categories_operation,
+                    get_monitor_events_operation, get_monitor_tenant_events_operation,
+                    get_monitor_metric_definitions_operation, get_monitor_metrics_operation,
+                    ServiceGroup, create_service_adapter)
+
+# MANAGEMENT COMMANDS
+alert_rules_operations = create_service_adapter(
+    'azure.cli.command_modules.monitor.sdk.operations.alert_rules_operations',
+    'AlertRulesOperations')
```

Comment:
I see but as I am following the flow/structure from `azure-cli-sql` (as I have understood that flow as of now :)), I'd prefer to keep it same. What's your thought on that?